configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionRepository.java

```diff
@@ -206,15 +206,13 @@ private boolean isActiveSession(LocalSession candidate) {
 
     public void deleteLocalSession(LocalSession session) {
         long sessionId = session.getSessionId();
-        try (Lock lock = lock(sessionId)) {
```

Comment:
Why was taking the lock removed here?

Reply:
I mistakenly thought the lock was not reentrant, so I think it is OK to add it back again